encoding/da.go

```diff
@@ -446,6 +436,27 @@ func ConstructBatchPayloadInBlob(chunks []*Chunk, MaxNumChunks uint64) ([]byte,
 	return batchBytes, nil
 }
 
+// GetKeccak256Gas calculates the gas cost for computing the keccak256 hash of a given size.
+func GetKeccak256Gas(size uint64) uint64 {
```

Comment:
Also for these helper functions: if not necessary for them to be public then they should be private to keep it simpler. 

Otherwise, I think this method specifically belongs to an `estimate` package maybe?